src/criteria/default/devices.js

```diff
@@ -0,0 +1,20 @@
+import {setMatchingFunction} from '../utils';
+
+setMatchingFunction('devices', ( config, ras, { optionParams } )  => {
+	const selectedDevices = Array.isArray(config.value) ? config.value : [];
+	if (selectedDevices.length === 0) {
+		return false;
+	}
+
+	const width = window.innerWidth;
+
+	return selectedDevices.some(deviceType => {
+		const device = optionParams[deviceType];
+		if (!device || !device.min_width || !device.max_width) {
```

Comment:
The `Mobile small` device will always fail this check, because its `min_width` is `0` and `0` is falsy. The check should rather be:

```suggestion
		if (!device || isNaN(device.min_width) || isNaN(device.max_width)) {
```

Reply:
Yeah, good call! Fixed.